README.md

```diff
@@ -23,6 +23,15 @@ Clojure PDF manipulation library & wrapper for [PDFBox](http://pdfbox.apache.org
 (pdf/merge-pdfs :input ["pdfs/clojure-1.pdf" "pdfs/clojure-2.pdf"] :output "foo.pdf")
 ```
 
+### Split a PDF into mutliple PDDocuments
+```clojure
+ (require '[pdfboxing.split :as pdf])
```

Comment:
Thanks for all the feedback. I will make some adjustments before I submit
again.

On Thu, May 28, 2015 at 8:47 AM, Александар Симић <notifications@github.com>
wrote:

> In README.md
> <https://github.com/dotemacs/pdfboxing/pull/10#discussion_r31234057>:
>
> > @@ -23,6 +23,15 @@ Clojure PDF manipulation library & wrapper for [PDFBox](http://pdfbox.apache.org
> >  (pdf/merge-pdfs :input ["pdfs/clojure-1.pdf" "pdfs/clojure-2.pdf"] :output "foo.pdf")
> >  ```
> >
> > +### Split a PDF into mutliple PDDocuments
> > +```clojure
> > + (require '[pdfboxing.split :as pdf])
>
> Thank you for writing the documentation, much appreciated!
>
> One thing I'd ask is if maybe instead of having the comments inline like
> you've added them, having them on their own above the code sample, e.g:
> Split a PDF into mutliple PDDocuments
>
> (require '[pdfboxing.split :as pdf])
>
> List of PDDocument pages 1 through 8
>
> (pdf/split-pdf :input "pdfs/clojure.pdf" :start 1 :end 8)
>
> Splits the pdf into single pages as a list of PDDocument
>
> (pdf/split-pdf :input "pdfs/clojure.pdf")
>
> Splits the pdf in half and writes them to disk as clojure-1.pdf and
> clojure-2.pdf
>
> (pdf/split-pdf-at :input "pdfs/clojure.pdf")
>
> Splits into two pdfs the first having 5 pages, second has rest
>
> (pdf/split-pdf-at :input "pdfs/clojure.pdf" :split 5)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotemacs/pdfboxing/pull/10/files#r31234057>.
>